ui/src/VoxettaMain.js

```diff
@@ -37,3 +37,36 @@ export class VoxettaMain extends LitElement {
 }
 
 customElements.define('voxetta-main', VoxettaMain);
+
+
+navigator.mediaDevices.getUserMedia({ audio: true, video: false })
+    .then(function(stream) {
+
+        const recordButton = document.getElementById("recordButton");
```

Comment:
FYI you can auto lint and format your code to the Google open source style guide with the scripts `npm run lint` and `npm run format`